File: two_d _array/row_max1.py

def row_max1(arr,m,n):
    max_count=0
    index=-1
    for i in range(m):
        current_count=0
        for j in range(n):
            if arr[i][j]==1:
                current_count+=1
        if max_count<current_count:
            index=i
            max_count=current_count
    return index

arr=[[1,0,1],[0,0,0],[1,0,1]]
row_index=row_max1(arr,len(arr),len(arr[0]))
print("row index has max 1=",row_index)

File: two_d _array/row_max1.java
public class row_max1
{
    public static void main(String args[])
    {
        int arr[][]={{1,0,1},{1,0,0},{1,1,1}};
        System.out.println("Array elements are:=");
        int max_count=0;
        int index=-1;
        for(int i=0;i<arr.length;i++)
        {
            int current_count=0;
            for(int j=0;j<arr[0].length;j++)
            {
                if(arr[i][j]==1)
                {
                    current_count++;
                }     
            }
            if(max_count<current_count)
                {
                    index=i;
                    max_count=current_count;
                }
        }
        System.out.println("row index have max no of 1="+index);
    }

}